crystal/distribution.yaml

```diff
@@ -1750,6 +1750,22 @@ repositories:
       url: https://github.com/ros-visualization/rqt_top.git
       version: crystal-devel
     status: maintained
+  rqt_topic:
+    doc:
+      type: git
+      url: https://github.com/ros-visualization/rqt_topic.git
+      version: crystal-devel
+    release:
+      tags:
+        release: release/crystal/{package}/{version}
+      url: https://github.com/ros2-gbp/rqt_topic-release.git
+      version: 1.0.0-0
+    source:
+      test_pull_requests: true
```

Comment:
The repository does neither contain anything to compile nor tests so a PR job seems unnecessary.

Reply:
good point. I was following the rqt example